libraries/botbuilder-ai/tests/luisRecognizer.test.js

```diff
@@ -115,7 +115,7 @@ function TestJson(file, done, includeAllIntents, includeInstance, telemetryClien
     var newPath = expectedPath + ".new";
     var context = new TestContext({ text: expected.text });
     var recognizer = new LuisRecognizer({ applicationId: luisAppId, endpointKey: endpointKey }, 
-        { includeAllIntents: includeAllIntents, includeInstanceData: includeInstance }, true, telemetryClient, logPersonalInformation);
+        { includeAllIntents: includeAllIntents, includeInstanceData: includeInstance, telemetryClient: telemetryClient, logPersonalInformation: logPersonalInformation }, true, telemetryClient, logPersonalInformation);
```

Comment:
It looks like you moved telemetry & PII into the options argument, which is great!
But shouldn't we also remove the extra `telemetryClient` and `logPersonalInformation` that comes after "`true`" (the `includeApiResults` parameter)?

Reply:
Awesome!  Thanks for the feedback!